src/search/lp/lp_internals.cc

```diff
@@ -118,12 +118,12 @@ unique_ptr<OsiSolverInterface> create_lp_solver(LPSolverType solver_type) {
         break;
     case LPSolverType::CPLEX:
 #ifdef COIN_HAS_CPX
-        {
-            OsiCpxSolverInterface *cpx_solver = new OsiCpxSolverInterface;
-            CPXsetintparam(cpx_solver->getEnvironmentPtr(), CPX_PARAM_THREADS, 1);
-            cpx_solver->passInMessageHandler(new ErrorCatchingCoinMessageHandler);
-            lp_solver = cpx_solver;
-        }
+    {
```

Comment:
Are these braces still necessary? I think they were a hack around the old version of uncrustify. If so, we should remove them at this point.